Client/Frontend/Reader/ReaderMode.js

```diff
@@ -119,6 +119,23 @@ var _firefox_ReaderMode = {
             }
         }
     },
+
+    // Loop over the style sheets, find all the the @font-face declarations and fix the
+    // relative src attributes so that they point to our embedded web server.
+    fixFontFaces: function(webServerBase) {
```

Comment:
Rather than doing this at runtime, is there any reason we can't choose a fixed port to run our webserver on, then add localhost:<port> directly to the stylesheets?